apps/callscreen/style/conference_group_ui.css

```diff
@@ -25,7 +25,12 @@ html[dir="rtl"] #group-call-details-list > section .additionalContactInfo {
   overflow: hidden;
   text-overflow: ellipsis;
   font-weight: bold;
-  -moz-padding-end: 4.5rem;
+  padding-right: 8rem;
+}
+html[dir="rtl"] #group-call-details-list .number bdi {
+  text-align: right;
```

Comment:
If it's not there, it's left aligned.  I inspected and I didn't see anything obvious overriding it, so it seems like we don't get it automatically.